core/src/stores/postgres.rs

```diff
@@ -89,10 +89,6 @@ impl Store for PostgresStore {
         let mut c = pool.get().await?;
         let tx = c.transaction().await?;
 
-        // Block executions, Runs & Runs joins: we execute a SQL function
-        tx.execute("SELECT delete_project_runs($1)", &[&project_id])
```

Comment:
Can we also remove the procedure `delete_project_runs`?

Reply:
I kept it at first but happy to remove it.

Reply:
Done!

Reply:
Getting rid of the dark magic! thank you 🙏🏻